pagopa/pn-frontend-e2e
# PN-9228-e-2-e-mittente-sezione-api-key

## Short description

The test **PN-9228** didn't have the last step defined in the [confluence page mittente mapping test](https://pagopa.atlassian.net/wiki/spaces/PN/pages/868778282/Mittente+-+Mapping+test-scenari).

## List of changes proposed in this pull request

- added new feature step `And Si copia correttamente la Api Key cliccando sul bottone di copia`
- added new feautre step `And Nella pagina Api Key posizionare il cursuore sopra il numero gruppi`
- updated webelements and methods related to the tests 

## How to test
Simply run `mvn clean test -Dcucumber.filter.tags=@TA_MittenteVisualizzazioneApiKey` and verify if the test works fine.
